### DISCUSSION TITLE
How to upload file to an image table and save the id to a product table when adding product to database in filament

### DISCUSSION THREAD

**Discussion by abdulsalamamtech** [filamentphp/filament#12544] [Help]
### Package

Form builder

### Package Version

v3.0

### How can we help you?

I'm trying to save an image to a database and get the image id and save it to the product table in filament.

This is how it will appear without using filament a need the same result while using filament:
```php

        $image = Image::create([
            'path' => '/product-2432432.png'
        ]);
        $product = Product::create([
            'name' => $request->name,
            'price' => $request->price,
            'image_id' => $image->id,
        ]);
```

you can also reach out to me on: abdulsalamamtech@gmail.com

Thanks!

**Comment by leandrocfe** [comment]
Maybe Spatie Media Library?
https://filamentphp.com/plugins/filament-spatie-media-library
